Source/WebCore/page/writing-tools/WritingToolsTypes.h

```diff
@@ -111,6 +111,15 @@ struct TextSuggestion {
     State state;
 };
 
+#pragma mark - Animations
+
+enum class RunAnimation : uint8_t {
```

Comment:
The build failures are because `RunAnimation` is inside `ENABLE(WRITING_TOOLS)`, but is used outside of it in other places.

Reply:
I also think it's slightly incorrect to have it in here at all, since the animations aren't just for Writing Tools.

Reply:
+1